Fixed servers with unknon versions failing to start if the discord br…

robinuniverse opened the pull request:
…idge was enabled

codieradical commented on src/gg/codie/mineonline/server/MinecraftServerLauncher.java:
```diff
@@ -55,8 +57,20 @@ public MinecraftServerLauncher(String[] args) throws Exception {
+        if (minecraftVersion != null)
+            ver = minecraftVersion.name;
+        else
+            ver = "";
```
I think this would be better as null rather than an empty string. You'd have to check it in the DCB constructor.
As in
String versionName = Minecraft version != null ? minecraftVersion.name : null;

codieradical commented on src/gg/codie/mineonline/server/MinecraftServerLauncher.java:
```diff
@@ -7,6 +7,7 @@
 import gg.codie.mineonline.discord.MessageRecievedListener;
 import gg.codie.mineonline.utils.Logging;
 import gg.codie.utils.MD5Checksum;
+import org.lwjgl.Sys;
```
Fairly sure this import is unused. If it isn't, it should be. Please remove it.

robinuniverse commented:
- Fixed unix alt tabbing crash bug
- Fixed server crashing if version is unknown and discord bridge is enabled
- Made the classic check more reliable
- Messages from discord now have sanity checks to make sure they can't crash classic players
- Minecraft compatible emojis are replaced with tags that codies client code can recognize, and has color in classic
 - Prevent empty messages from getting through (images, embeds, empty codeblocks)

robinuniverse commented:
(also made requested changes)
